### PR TITLE
remove ip package

### PR DESCRIPTION
**Pull request by JoshGlazebrook**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "socks5"
   ],
   "engines": {
-    "node": ">= 10.13.0",
+    "node": ">= 16.0.0",
```
Comment (G-Rath) [inline package.json:26]: dropping support for node version is a breaking change - it looks like `ip-address` v7 supports Node v10; would it be possible to use that instead?

**Comment by G-Rath** [comment]
just want to say thanks for the fast response on this ❤️

File: package.json (hunk 1 above)
Comment (JoshGlazebrook) [reply package.json:26]: I think i'll just bump the overall package to 3.0.0. Anything prior to node 16.x isn't being maintained at this point.

Comment (G-Rath) [reply package.json:26]: that'll mean downstream packages will need to explicitly update their dependency on `sock` which could delay rollout - any chance of getting a final v2 patch with this before immediately releasing v3?

Comment (G-Rath) [reply package.json:26]: btw If you're running into problems related to Node 10, I'm happy to have a crack at getting things working :)

Comment (JoshGlazebrook) [reply package.json]: I think it may just be the typescript building not working on 10.x, the actual javascript may work fine tbh.

Comment (JoshGlazebrook) [reply package.json]: Okay, yeah so it works just fine on plain javascript node 10.24.1 with ip-address 9.x, it's just the building of the typescript/etc won't work for anything less than 16.

Comment (JoshGlazebrook) [reply package.json]: So I'll just leave the min version as-is with 10.x, and release as 2.8.0? Or should this just be a bug/patch as 2.7.2?

Comment (G-Rath) [reply package.json]: I think technically it's a feature but I don't think anyone would complain if you released it as a bug fix 🤷

**Comment by retroburst** [comment]
Thank you for sorting this out so quickly, much appreciated!

**Pull request merged**